llvm/lib/TargetParser/SubtargetFeature.cpp

```diff
@@ -81,5 +82,7 @@ void SubtargetFeatures::getDefaultSubtargetFeatures(const Triple& Triple) {
       AddFeature("64bit");
       AddFeature("altivec");
     }
+  } else if (Triple.isRISCV64() && TargetABI.contains("lp64d")) {
```

Comment:
Should we extend the checks here for both 32-bit and other hard floats?

- ilp32f
- ilp32d
- lp64f
- lp64q